srcpkgs/nvidia-vaapi-driver/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'nvidia-vaapi-driver'
+pkgname=nvidia-vaapi-driver
+version=0.0.7
+revision=1
+archs="x86_64"
+build_style=meson
+hostmakedepends="pkg-config libglvnd-devel libva-devel gst-plugins-bad1-devel"
+makedepends="nv-codec-headers"
+depends="nvidia gst-plugins-bad1"
```

Comment:
This whole line can be removed; `gst-plugins-bad1` gets auto-detected as well as `libEGL`, one you didn't explicitly list.  `nvidia` shouldn't be a hard-requirement, as there is a plan to also package `nvidia-open` at some point.

```
=> nvidia-vaapi-driver-0.0.7_1: running pre-pkg hook: 999-collected-rdeps ...
   glibc>=2.32_1 gst-plugins-bad1>=1.18.3_2 libEGL>=7.11_1
```